README.md

```diff
@@ -203,6 +203,25 @@ public class DroidContractConfiguration extends ContractConfiguration<DroidContr
 }
 ``` 
 
+Also, imported contracts can be wrapped with `com.joom.lightsaber.Lazy` or `kotlin.Lazy`. In this case contract will be instantiated only when some of its dependencies would need. It can reduce `ContractConfiguration` creation time, especially for contracts with large amount of dependencies. 
```

Comment:
`In this case contract` -> `In this case, the contract`?
`large amount of` -> `many` ?

Reply:
fixed